src/main/java/uk/gov/hmcts/darts/common/entity/ExternalObjectDirectoryEntity.java

```diff
@@ -87,6 +87,9 @@ public class ExternalObjectDirectoryEntity extends CreatedModifiedBaseEntity imp
     @Column(name = "is_response_cleaned")
     private boolean responseCleaned;
 
+    @Column(name = "update_retention")
+    private boolean updateRetention;
```

Comment:
some entities are using Boolean - others boolean - do we know which one is preferred or whether it matters?